soc/arc/snps_nsim/Kconfig

```diff
@@ -9,6 +9,7 @@ choice
 config SOC_NSIM_EM
 	bool "Synopsys ARC EM in nSIM"
 	select CPU_HAS_MPU
+	select CPU_HAS_DSP
```

Comment:
I think we should introduce a new nSIM target for DSP, to ensure we have coverage with and without DSP extensions.

Reply:
Will add it and test case in next commit